Source/Diagnostics/BTDiagnostics.H

```diff
@@ -375,26 +375,26 @@ private:
     /** Interleave lab-frame meta-data of the buffers to be consistent
      *  with the merged plotfile lab-frame data.
      */
-    void InterleaveBufferAndSnapshotHeader ( std::string buffer_Header,
-                                             std::string snapshot_Header);
+    void InterleaveBufferAndSnapshotHeader (const std::string& buffer_Header,
+                                            const std::string& snapshot_Header);
     /** Interleave meta-data of the buffer multifabs to be consistent
      *  with the merged plotfile lab-frame data.
      */
-    void InterleaveFabArrayHeader (std::string Buffer_FabHeader_path,
-                                   std::string snapshot_FabHeader_path,
-                                   std::string newsnapshot_FabFilename);
+    void InterleaveFabArrayHeader(const std::string& Buffer_FabHeader_path,
```

Comment:
This is missing the space before the parenthesis
```suggestion
    void InterleaveFabArrayHeader (const std::string& Buffer_FabHeader_path,
```

Reply:
Though I see many other places without the space, so probably doesn't need to be fixed. It's a nice detail but used inconsistently.